translations/en.json

```diff
@@ -314,7 +314,8 @@
     "mailserver-automatic": "Automatic selection",
     "currency-display-name-kzt": "Kazakhstan Tenge",
     "no-recent-chats": "There are no recent chats here yet. \nUse the (+) button to discover people \nto chat with",
-    "install": "Install",
+    "install": "↓ Install",
```

Comment:
i'm not sure if we should use an image here @errorists ?

Reply:
image you mean the downward arrow? I think we should, why not?

Reply:
@flexsurfer means that here the arrow is an emoticon char not an image

Reply:
downward arrow yes, do we have an image we can use ? i found only this one 
![image](https://user-images.githubusercontent.com/11790366/64855326-ef61b900-d61f-11e9-845a-96204d932140.png)

Reply:
if we have the choice it's actually better to use a character in the font, since it will ideally match the weight and size of the text next to it, producing a more visually pleasing and consistent look. It's actually what Apple did in iOS13, all icons were replaced by unicode characters in font files. [Apple Developer](https://developer.apple.com/design/human-interface-guidelines/sf-symbols/overview/)

Reply:
currently, we're limited to arrows as that's what provided in the font, but it's open source and we could expand it with all of our own icons if we'd wanted to. 
To sum up, if we're about to place any of these ← → ↙ ↘ ↖ ↗ ↑ ↓ ↪ ↩ ✓ ✗ next to text, it's better to just use a character and not an image :)